.devcontainer/devcontainer.json

```
@@ -0,0 +1,64 @@
+{
+    "name": "prototyping",
+    "build": {
+        "dockerfile": "Dockerfile"
+    },
+    "forwardPorts": [3000],
+    "initializeCommand": "bash .devcontainer/initializeCommand.sh",
+    "onCreateCommand": "bash .devcontainer/onCreateCommand.sh",
+    "updateContentCommand": "bash .devcontainer/updateContentCommand.sh",
+    "postStartCommand": "git config --global --add safe.directory ${containerWorkspaceFolder}",
+    "containerEnv": {
+        "WORKSPACE": "${containerWorkspaceFolder}",
+        "XDG_CONFIG_HOME": "/home/gitpod/.config"
+    },
+    "features": {
+        "ghcr.io/devcontainers/features/git:1": {},
+        "ghcr.io/devcontainers/features/github-cli:1": {},
+        "ghcr.io/devcontainers/features/docker-in-docker:2": {},
```

Just found a memory leak affecting all our devcontainers. Please change this line

```suggestion
        "ghcr.io/devcontainers/features/docker-in-docker:2": {
            "dockerDashComposeVersion": "none",
            "installDockerComposeSwitch": false
        },
```